mycroft/client/enclosure/api.py

```diff
@@ -146,6 +170,9 @@ def eyes_volume(self, volume):
         Args:
             volume (int): 0 to 11
         """
+        if volume < 0 or volume > 11:
+            raise ValueError('volume ({}) must be between 0-100'.
```

Comment:
between 0 and 11?

Reply:
@penrods After that change, it looks good to go.